src/layers/GridLines.tsx

```diff
@@ -30,13 +30,212 @@ export const GridLines = ({ height, domain, smallerZoomScale, timeScale, weekStr
+    case ZoomLevels.THREE_HOURS:
+      return <MinutesView height={height} domain={domain} timeScale={timeScale} ones={true} halves={false} quarters={false} />
+    case ZoomLevels.ONE_HOUR:
+      return <MinutesView height={height} domain={domain} timeScale={timeScale} ones={true} halves={true} quarters={false} />
+    case ZoomLevels.THIRTY_MINS:
+      return <MinutesView height={height} domain={domain} timeScale={timeScale} ones={true} halves={true} quarters={true} />
     default:
       return (
-        <MonthView height={height} domain={domain} timeScale={timeScale} showWeekStripes={weekStripes === undefined ? true : weekStripes} />
+        <BoundsView height={height} domain={domain} timeScale={timeScale} />
       )
   }
 }
 
+interface TickLineProps {
+  xPosition: number
+}
+interface HourTickViewProps {
+  height: number
+  domain: [number, number]
+  timeScale: ScaleLinear<number, number>
+  halves?: boolean
+  quarters?: boolean
+  eights?: boolean
+}
+
+interface MinuteTickViewProps {
+  height: number
+  domain: [number, number]
+  timeScale: ScaleLinear<number, number>
+  ones?: boolean
+  halves?: boolean
+  quarters?: boolean
+}
+
+const SmallTickLine = ({ xPosition }: TickLineProps) => {
+  const xAxisTheme = useTimelineTheme().xAxis
+  const classes = useHourViewStyles(xAxisTheme)
+  return (
+    <line
+      className={classes.line}
+      x1={xPosition}
+      y1={0}
+      x2={xPosition}
+      y2={'95%'}
+      strokeWidth={.5}
+    />
+  )
+}
+
+const DAY_MS = 24 * 60 * 60 * 1000
+const TWELVE_HOURS_MS = DAY_MS / 2
+const SIX_HOURS_MS = TWELVE_HOURS_MS / 2
+const THREE_HOURS_MS = SIX_HOURS_MS / 2
+const TWO_HOURS_MS = SIX_HOURS_MS / 3
+const HOURS_MS = TWO_HOURS_MS / 2
+const HALF_HOURS_MS = HOURS_MS / 2
+const QUARTER_HOURS_MS = HALF_HOURS_MS / 2
+
+/* ·················································································································· */
+/*  Hours 
+/* ·················································································································· */
+const HoursView = ({ height, domain, timeScale, halves, quarters, eights }: HourTickViewProps) => {
+  const xAxisTheme = useTimelineTheme().xAxis
+  const classes = useHourViewStyles(xAxisTheme)
+  const leftBoundMs = domain[0] - (domain[0] % THREE_HOURS_MS)
+  const rightBoundMs = domain[1]
+
+  let dayTicks = []
+  let halfDayTicks = []
+  let quarterDayTicks = []
+  let eightDayTicks = []
+  // Set up the tick marks based off 24 hours, 12 hours, 6 hours, 3 hours
+  for (let time = leftBoundMs; time < rightBoundMs; time += THREE_HOURS_MS) {
+    if (time % DAY_MS === 0) {
+      dayTicks.push(time);
+    } else if (time % TWELVE_HOURS_MS === 0) {
+      halfDayTicks.push(time);
+    } else if (time % SIX_HOURS_MS === 0) {
+      quarterDayTicks.push(time)
+    } else {
+      eightDayTicks.push(time)
+    }
+  }
+
+  const dayLines = dayTicks.map(time => {
+    const x = timeScale(time)!
```

Comment:
Can we break out this logic on its own? Currently it seems to be doing something similar in every map.

So we could get it down to something like:

```
const SmallLine = (x, y, className) => {
  return (
    <g>
      <SmallTickLine xPosition={x} />
      <text className={className} x={x} y={y}>
        {new Date(time).toLocaleTimeString()}
      </text>
    </g>
  );
};

const HoursView= () => {
....
      const dayLines = dayTicks.map(time => SmallLine(timeScale(time)!, height, classes.label));
      const halfDayLines = halves
           ? halfDayTicks.map(time => SmallLine(timeScale(time)!, height, classes.label))
            : [];
...
};
```